repo.sh

```diff
@@ -59,12 +59,12 @@ _checkout ()
 
         # If a directory exists and it is nonempty, assume the repo has been cloned.
         cd "${DEVSTACK_WORKSPACE}"  # Appsembler: Just in case `customer_specific` had an error.
-        if [ "$name" != "edx-theme-customers" -a -d "$name" -a -n "$(ls -A "$name" 2>/dev/null)" ]; then
+        if [ "$name" != "edx-theme-customers" ] && [ -d "$name" ] && [ -n "$(ls -A "$name" 2>/dev/null)" ]; then
```

Comment:
Is this just a readability improvement or is there a functional difference between the `-a` and `&&` for logical ands?

Reply:
I did it because PyCharm complained that `-a` is not well defined and advised me to use `[ ] && [ ]` which something I found also more readable.

So it's more cosmetic than functional.

Reply:
If we really want to get into it, my understanding is that `[` is obsolete and should pretty much always be replaced with `[[` now: http://mywiki.wooledge.org/BashFAQ/031

I kind of feel like once it gets to the point where that kind of thing comes up, we might want to think about just porting from bash to python.

Reply:
No I don't want to get into this :sweat_smile: I'm getting it back as it was. I can see that the [BashFaq](http://mywiki.wooledge.org/BashFAQ/031) and [Shell Check](https://github.com/koalaman/shellcheck/wiki/SC2166) are pointing to different directions, so I'm opting out to original status.